cmd/controller/main.go

```diff
@@ -91,6 +92,7 @@ func main() {
 		terminator,
 		node.NewController(manager.GetClient()),
 		metrics.NewController(manager.GetClient(), cloudProvider),
+		counter.NewController(ctx, manager.GetClient()),
```

Comment:
Can you create an issue to fold this into the metrics controller?m

Reply:
https://github.com/awslabs/karpenter/issues/840